### DISCUSSION TITLE
Transaction API error: Transaction already closed: Transaction is no longer valid. Last state: 'Expired'.

### DISCUSSION THREAD

**Discussion by deepajais95** [prisma/prisma#9825] [Q&A]
why i am getting this error while using transaction??

**Comment by ryands17** [comment]
@deepajais95 👋 

Could you share the query you're making and when does this error occur?

**Reply by deepajais95** [reply]
{
    maxWait: 2000, 
    timeout: 5000, 
  }
this is happening because of default value of time out, when i have changed this to 1 mint then it is working

**Reply by ryands17** [reply]
@deepajais95 👋 

I checked and this is a known issue at the moment. Please follow this issue for more updates: https://github.com/prisma/prisma/issues/8707

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
